### PR TITLE
Better Footer with social links #3311

### PR DESCRIPTION
**Pull request by SURYANSH-RAI**
Updated the followings
  1. added footer on webpage with social links as suggested in #3311
  2. added images related to social links in images folder
  3. updated Readme.md:-
        -> centred the heading for a more soothing look
        -> added table of contents with link for better navigation on page
        -> updated UI related things like managing spaces between lines, and updating appearance of "impressions" content
        -> added a link of your mail in support section
  4. deleted mail.jpg as now a link to mail is there
  5. **Note:- the link for donations was taken from Readme.md and no changes in that link is there**

### CODE REVIEW

**Comment by jomjol** [comment]
@SURYANSH-RAI Thanks for the updates. I'm fine with them beside one change: change my email from an image to a direct link.

I use a picture by purpose, so that e.g. robots do not screen my mail diretly. If somebode wants to contact me, he still needs to type it. So if you could remove that from the footer and the readme.md, I'm fine with the changes.

**Comment by SURYANSH-RAI** [comment]
@jomjol Sure i will do that. That will be a pleasure of mine.

**Comment by caco3** [comment]
@SURYANSH-RAI It looks like as if you didn't yet change it.

**Comment by SURYANSH-RAI** [comment]
Sorry Brother, there were some exams of mine. Surely i will update it today only.

File: README.md [hunk 1]
```diff
@@ -129,37 +180,46 @@ If you'd like to support the developer with a cup of coffee, you can do so via [
...
+<br>
+
 ## Support 💬
 If you have any technical problems, please search the [discussions](https://github.com/jomjol/AI-on-the-edge-device/discussions). In case you find a bug or have a feature request, please open an [issue](https://github.com/jomjol/AI-on-the-edge-device/issues).
 
-For any other issues, you can contact the developer via email:  
-<p align="center">
-  <img src="https://raw.githubusercontent.com/jomjol/AI-on-the-edge-device/master/images/mail.jpg" height="25">
-</p>
+For any other issues, you can contact the developer via email: mueller.josef@gmail.com
```
Comment (caco3) [inline README.md:188]:
```suggestion
For any other issues, you can contact the developer via email:  
<p align="center">
  <img src="https://raw.githubusercontent.com/jomjol/AI-on-the-edge-device/master/images/mail.jpg" height="25">
</p>
```

File: sd-card/html/index.html [hunk 2]
```diff
@@ -146,6 +170,28 @@ <h2>An ESP32 all inclusive neural network recognition system for meter Digitizat
 
 <span id="Version" style="font-size: 10px; margin-top: -5px;padding-left: 10px;">Loading version...</span>
 
+<div class="footer">
+    <div class="footer-section">
+       <span>Support & Contact Us</span>
+       <a href="https://github.com/jomjol/AI-on-the-edge-device" target="_blank" title="GitHub">
+          <img src="https://github.com/jomjol/AI-on-the-edge-device/images/github-logo.png" alt="GitHub">
+       </a>
+       <a href="mailto:mueller.josef@gmail.com" title="Email Support">
```
Comment (caco3) [inline sd-card/html/index.html:179]:
```suggestion
```

File: docs/index.html [hunk 3]
```diff
@@ -34,5 +57,28 @@ <h2>Notes:</h2>
...
       <hr>
       <p style="font-size: small;">Installer and Console powered by <a href=https://esphome.github.io/esp-web-tools/ target=_blank>ESP Web Tools</a></p>
+
+   <div class="footer">
+      <div class="footer-section">
+       <span>Support & Contact Us</span>
+       <a href="https://github.com/jomjol/AI-on-the-edge-device" target="_blank" title="GitHub">
+          <img src="https://github.com/jomjol/AI-on-the-edge-device/images/github-logo.png" alt="GitHub">
+       </a>
+       <a href="mailto:mueller.josef@gmail.com" title="Email Support">
```
Comment (caco3) [inline docs/index.html:67]:
```suggestion
```

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (SURYANSH-RAI) [reply README.md]: Yes Sir

**Pull request merged**